test/cache_test.rb

```diff
@@ -4,6 +4,26 @@
 
 module ActiveModelSerializers
   class CacheTest < ActiveSupport::TestCase
+    class AuthorWithCustomCacheKey < Author
```

Comment:
I like that last one, I'll change pr to use that

On Aug 12, 2016 8:25 PM, "Benjamin Fleischer" <notifications@github.com>
wrote:

> In test/cache_test.rb
> <https://github.com/rails-api/active_model_serializers/pull/1878#discussion_r74628088>
> :
>
> > @@ -4,6 +4,26 @@
> >
> >  module ActiveModelSerializers
> >    class CacheTest < ActiveSupport::TestCase
> > +    class AuthorWithCustomCacheKey < Author
>
> or AuthorWithExpandableCacheKeyElements?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1878/files/4ce7dc06280eb3b9580fab21956ce1202919ca5c#r74628088>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADRa4mGlYcKqJZ9u1mlwJC-vfdol4u_ks5qfKyFgaJpZM4Ji8cj>
> .
>